src/main/java/frc/robot/subsystems/tank/TankSubsystem.java

```diff
@@ -212,6 +220,17 @@ public DifferentialDriveWheelSpeeds getWheelSpeeds() {
       rightEncoder.getVelocity());
   }
 
+  public void setCurrentLimit(int limit) {
+    this.currentLimit = limit;
+
+    int motorLimit = currentLimit/4; //number of motors
```

Comment:
Be careful with integer division. Do you know what happens for `currentLimit` not being evenly divisible by 4?